quobyte.go

```diff
@@ -24,44 +25,40 @@ func NewQuobyteClient(url string, username string, password string) *QuobyteClie
 	return result
 }
 
-type CreateVolumeRequest struct {
-	Name        string `json:"name"`
-	RootUserId  string `json:"root_user_id"`
-	RootGroupId string `json:"root_group_id"`
-}
-
-type CreateVolumeResponse struct {
-	VolumeUuid string `json:"volume_uuid"`
-}
-
 // Create a new Quobyte volume. Its root directory will be owned by given user and group
 func (client QuobyteClient) CreateVolume(name string, rootUserName string, rootGroupName string) (volumeUuid string, err error) {
-	request := &CreateVolumeRequest{
+	request := &createVolumeRequest{
 		Name:        name,
-		RootUserId:  rootUserName,
-		RootGroupId: rootGroupName,
+		RootUserID:  rootUserName,
+		RootGroupID: rootGroupName,
 	}
-	var response CreateVolumeResponse
-	err = client.sendRequest("createVolume", request, &response)
-	if err != nil {
+	var response createVolumeResponse
+	if err = client.sendRequest("createVolume", request, &response); err != nil {
 		return "", err
 	}
-	return response.VolumeUuid, nil
-}
 
-type DeleteVolumeRequest struct {
-	VolumeUuid string `json:"volume_uuid"`
+	return response.VolumeUUID, nil
 }
 
-type DeleteVolumeResponse struct {
+// ResolveVolumeNameToUUID resolves a volume name to a UUID
+func (client QuobyteClient) ResolveVolumeNameToUUID(volumeName string) ([]string, error) {
+	request := &resolveVolumeNameRequest{
+		VolumeName: volumeName,
+	}
+	var response resolveVolumeNameResponse
+	if err := client.sendRequest("createVolume", request, &response); err != nil {
+		return []string{}, err
+	}
+
+	return response.VolumeUUID, nil
 }
 
 // Delete a Quobyte volume. Its root directory will be owned by given user and group and have access 700.
-func (client QuobyteClient) DeleteVolume(volumeUuid string) error {
-	request := &DeleteVolumeRequest{
-		VolumeUuid: volumeUuid,
+func (client QuobyteClient) DeleteVolume(volumeUUID string) error {
```

Comment:
Added, it's a nice feature!

BTW.: I pushed my initial commit on a golang docker quobyte plugin https://github.com/johscheuer/go-quobyte-docker I will test it later this week.